tests/e2e/jest.config.js

```diff
@@ -7,6 +7,7 @@ module.exports = {
 		'<rootDir>/specs/**/__tests__/**/*.js',
 		'<rootDir>/specs/**/?(*.)(spec|test).js',
 		'<rootDir>/specs/**/test/*.js',
+		'<rootDir>/stories-editor/**/?(*.)(spec|test).js',
```

Comment:
Can we please keep all the tests within `tests/e2e/specs`? 🙂 

We can organize them under `tests/e2e/specs/stories-editor`.

Reply:
Yes, definitely :)

Reply:
Moved within 8118ac9.